pkgs/tools/misc/hackertyper/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, ncurses }:
+
+stdenv.mkDerivation rec {
+  pname = "hackertyper";
+  version = "02262019";
```

Comment:
Usually we go with YYYYMMDD since a format like MMDDYYYY may not always be an incrementing in number size, not that it matters that much for nix, it's more a convention.